prospero-cli/src/main/java/org/wildfly/prospero/cli/RepositoryDefinition.java

```diff
@@ -56,4 +69,31 @@ private static boolean isValidUrl(String text) {
             return false;
         }
     }
+
+    public static URI getAbsoluteFileURI(String text) throws ArgumentParsingException, URISyntaxException {
+        String path;
+        path = getPath(text);
+        File file = new File(path);
+        if (file.exists()) {
+            return file.toURI();
+        } else {
+            throw CliMessages.MESSAGES.invalidFilePath(path);
+        }
+    }
+
+    public static String getPath(String text) throws URISyntaxException {
+        if (text.startsWith("file")) {
```

Comment:
I think this should be `file:` to make sure the user is inputting a URI and not a file named `file_dir` for example